lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.AirLoopHVAC.rb

```diff
@@ -15,6 +15,130 @@ def air_loop_hvac_unoccupied_threshold
     return 0.05
   end
 
+  # Determine the economizer type and limits for the the PRM
+  # Defaults to 90.1-2007 logic.
+  #
+  # @param air_loop_hvac [OpenStudio::Model::AirLoopHVAC] air loop
+  # @param climate_zone [String] ASHRAE climate zone, e.g. 'ASHRAE 169-2013-4A'
+  # @return [Array<Double>] [economizer_type, drybulb_limit_f, enthalpy_limit_btu_per_lb, dewpoint_limit_f]
+  def air_loop_hvac_prm_economizer_type_and_limits(air_loop_hvac, climate_zone)
+    economizer_type = 'NoEconomizer'
+    drybulb_limit_f = nil
+    enthalpy_limit_btu_per_lb = nil
+    dewpoint_limit_f = nil
+
+    case climate_zone
+    when 'ASHRAE 169-2006-0B',
+      'ASHRAE 169-2006-1B',
+      'ASHRAE 169-2006-2B',
+      'ASHRAE 169-2006-3B',
+      'ASHRAE 169-2006-3C',
+      'ASHRAE 169-2006-4B',
+      'ASHRAE 169-2006-4C',
+      'ASHRAE 169-2006-5B',
+      'ASHRAE 169-2006-5C',
+      'ASHRAE 169-2006-6B',
+      'ASHRAE 169-2006-7A',
+      'ASHRAE 169-2006-7B',
+      'ASHRAE 169-2006-8A',
+      'ASHRAE 169-2006-8B',
+      'ASHRAE 169-2013-0B',
+      'ASHRAE 169-2013-1B',
+      'ASHRAE 169-2013-2B',
+      'ASHRAE 169-2013-3B',
+      'ASHRAE 169-2013-3C',
+      'ASHRAE 169-2013-4B',
+      'ASHRAE 169-2013-4C',
+      'ASHRAE 169-2013-5B',
+      'ASHRAE 169-2013-5C',
+      'ASHRAE 169-2013-6B',
+      'ASHRAE 169-2013-7A',
+      'ASHRAE 169-2013-7B',
+      'ASHRAE 169-2013-8A',
+      'ASHRAE 169-2013-8B'
+      economizer_type = 'FixedDryBulb'
+      drybulb_limit_f = 75
+    when 'ASHRAE 169-2006-5A',
+      'ASHRAE 169-2006-6A',
+      'ASHRAE 169-2013-5A',
+      'ASHRAE 169-2013-6A',
+      economizer_type = 'FixedDryBulb'
+      drybulb_limit_f = 70
+    else
+      economizer_type = 'FixedDryBulb'
+      drybulb_limit_f = 65
+    end
+
+    return [economizer_type, drybulb_limit_f, enthalpy_limit_btu_per_lb, dewpoint_limit_f]
+  end
+
+  # Determine if an economizer is required per the PRM.
+  #
+  # @param air_loop_hvac [OpenStudio::Model::AirLoopHVAC] air loop
+  # @param climate_zone [String] ASHRAE climate zone, e.g. 'ASHRAE 169-2013-4A'
+  # @return [Bool] returns true if required, false if not
+  def air_loop_hvac_prm_baseline_economizer_required?(air_loop_hvac, climate_zone)
+    economizer_required = false
+    baseline_system_type = air_loop_hvac.additionalProperties.getFeatureAsString('baseline_system_type').get
+    # System type 3 through 8 and 11, 12 and 13
+    if ['SZ_AC', 'PSZ_AC', 'PVAV_Reheat', 'VAV_Reheat', 'SZ_VAV', 'PSZ_HP', 'SZ_CV', 'PSZ_HP', 'PVAV_PFP_Boxes', 'VAV_PFP_Boxes'].include? baseline_system_type
+      case climate_zone
```

Comment:
I think we should check for climate zone match more generically, in case the climate zone sets change in the future. Start with a list of strings to check like econo_not_required = ['0A', '0B', '1A', '1B', '2A', '3A', '4A'].
Then loop through this list and check for a string match in the current climate_zone. I'm not sure that CZs 7 and 8 will always have an "A" and "B" vesion, so checking the not_required list is probably safer.

Reply:
Updated https://github.com/NREL/openstudio-standards/blob/economizer_exception_prm/lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.AirLoopHVAC.rb#L53